### PR TITLE
07-compiler-usage, added `--jvm`

### PR DESCRIPTION
**Pull request by uvtc**
I put it after `--java`, like in the output of `haxe --help`.

### CODE REVIEW

**Comment by uvtc** [comment]
The JVM section I added to the content/12-target-details.md doc is a lightly-edited copy/paste of the Java section. I put it after the Java section, same order as it is shown in the output of `haxe --help`.

**Comment by uvtc** [comment]
See also issue #459 .

File: content/12-target-details.md [hunk 1]
```diff
@@ -1945,6 +1945,52 @@ java -jar bin/Main.jar
...
+#### Getting started with Haxe/JVM
+
+To get started with Haxe/JVM, create a new folder and save this class as `Main.hx`.
+
+[code asset](assets/HelloWorld.hx)
+
+To compile Haxe to JVM bytecode we need two obvious prerequisites installed:
+
+* [hxjava haxelib](http://lib.haxe.org/p/hxjava). This is the support library for the Java backend of the Haxe compiler.
+* [JRE - Java Runtime Environment](https://java.com/download/).
```
Comment (RealyUniqueName) [inline content/12-target-details.md:1963]: I think this is not required to compile to jvm.

File: content/12-target-details.md [hunk 2]
```diff
@@ -1945,6 +1945,52 @@ java -jar bin/Main.jar
...
+
+Run the following from the command line:
+
+```hxml
+haxe --jvm bin/Main.jar --main Main
+```
+
+Another possibility is to create and run (double-click) a file called `compile.hxml`. In this example the hxml-file should be in the same directory as the example class.
```
Comment (RealyUniqueName) [inline content/12-target-details.md:1971]: > run (double-click)

This is not the case for all operating systems. AFAIK double-click is set up only in windows if Haxe was installed via the official installer.

File: content/12-target-details.md (hunk 1 above)
Comment (RealyUniqueName) [reply content/12-target-details.md:1963]: It's required to run a compiled jar.

Comment (uvtc) [reply content/12-target-details.md]: Ok, clarified this.

File: content/12-target-details.md (hunk 2 above)
Comment (uvtc) [reply content/12-target-details.md]: Ok. Left in a comment that some OS's let you double-click on a jar, though I don't think I've ever run a jar this way.

File: content/12-target-details.md [hunk 3]
```diff
@@ -1945,6 +1945,56 @@ java -jar bin/Main.jar
...
+
+to produce the bin/Main.jar file.
+
+To execute the jar file, you'll need to have a JRE (Java Runtime Environment) installed, such as [OpenJDK](https://openjdk.java.net/) or a commercially-supported JRE. Then run the jar file using the following command:
+
+```sh
+java -jar bin/Main.jar
+```
+
+Note that some operating systems may allow you to double-click a jar file to run it.
```
Comment (Simn) [inline content/12-target-details.md:1987]: That information is pretty random, I don't think we need that here.

Comment (uvtc) [reply content/12-target-details.md]: I Agree.

Comment (uvtc) [reply content/12-target-details.md]: Removed.

**Comment by Simn** [comment]
Thanks!

**Pull request merged**